binary-oracle-pair/program/src/instruction.rs

```diff
@@ -66,8 +68,8 @@ pub enum PoolInstruction {
     ///   5. `[w]` token_P PASS mint
```

Comment:
I didn't mention this explicitly, but this should have a user transfer authority as well, same as for deposit